app/main.js

```diff
@@ -89,6 +89,10 @@
 
     async function initTheme() {
         F.util.chooseTheme(await F.state.get('theme', 'default'));
+        const orgStylesheet = (await F.currentUser.getOrg()).attributes.custom_css;
```

Comment:
attributes is basically internal on Backbone models, use `get('custom_css')` instead.